### DISCUSSION TITLE
20220617

### DISCUSSION THREAD

**Discussion by tmiw** [ARSFI/paclink-open-source#28] [General]
This is a pre-production build intended to gauge current status of the project and determine what needs to be fixed. Brief testing indicates that packet (using Kenwood TM-D700A), AGW (using AGWPE and UZ7HO soundmodem) and Telnet channels work; Pactor is TBD.

Contains (vs. [last pre-release version](https://github.com/ARSFI/paclink-open-source/releases/tag/nightly_20220531)):

* Pactor: prevent frequency selection in connect dialog from being overridden.
* Pactor: prevent duplicate frequencies from being displayed (e.g. if the gateway also supports VARA or ARDOP).

Instructions:

1. Unzip paclink-open-source.zip somewhere on your hard drive.
2. Run setup.exe to install on your system. Paclink will be installed in the Start menu.
3. Open Paclink from the Start menu and reconfigure if required. (There is no import functionality from the existing Paclink at this time but if you used a pre-release version, configuration should be saved.)

Please leave comments in the release's discussion thread or email K6AQ@winlink.org with feedback. 👍
 

<hr /><em>This discussion was created from the release <a href='https://github.com/ARSFI/paclink-open-source/releases/tag/nightly_20220617'>20220617</a>.</em>

**Comment by kd4vrf** [comment]
I still see the duplicates in the connect dialog box and in the setup. I thought maybe I had installed the older version by mistake since nothing changed for me. I just got around to making sure all of the older copies of Paclink were removed from my VM and tried the install again to make sure I had the latest version. I still see the duplicates after confirming I have the latest build.

![Screen Shot 2022-06-30 at 9 53 15 PM](https://user-images.githubusercontent.com/45251519/176806988-11eec52d-b791-49b4-80ce-2dd09f00f6b3.png)

The issue with the first selection not working and having to repeat the selection for a second time is resolved for me in this build.

**Reply by tmiw** [reply]
I investigated further and I didn't initially notice the duplicates issue until I updated the channel list through Pactor setup. Turns out that the code wasn't cleaning up the old channel list during updates, causing duplicates to appear. The more you update the channel list, the more duplicates there'll be :)

Anyway, https://github.com/ARSFI/paclink-open-source/releases/tag/nightly_20220703 should resolve that issue. Note that you'll need to update the channel list one more time after installing this version for the duplicates to go away.

EDIT: fixed typo in URL.
